scripts/prepare-test-data.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+#
+# A script to download and extract test data.
```

Comment:
Please add a comment here telling people how to bootstrap:

* If they have a new test file which cannot be yet gotten by `wget`
* then they can put it here in the place `wget` _would have put it_
* then they can run their tests
* then if that works: hyperlink to the instructions for the `TIleDB-SOMA-Test-Data` repo